hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -146,12 +146,24 @@ public class DatanodeConfiguration {
    */
   @Config(key = "block.delete.queue.limit",
       type = ConfigType.INT,
-      defaultValue = "1440",
+      defaultValue = "5",
       tags = {DATANODE},
       description = "The maximum number of block delete commands queued on " +
           " a datanode"
   )
-  private int blockDeleteQueueLimit = 60 * 24;
+  private int blockDeleteQueueLimit = 5;
+
+  /**
+   * The maximum number of commands in queued list.
+   * if the commands limit crosses limit, then command will be ignored.
+   */
+  @Config(key = "command.queue.limit",
+      type = ConfigType.INT,
+      defaultValue = "5000",
+      tags = {DATANODE},
+      description = "The maximum number of commands queued on a datanode"
```

Comment:
This description isn't really accurate - its really max commands of each command type that can be queued on a datanode, right? As the commands go from the command queue into sub queues in the various handlers in many cases.

Reply:
Updated...